block/manager.go

```diff
@@ -43,6 +46,15 @@ const (
 	daBlock       blockSource = "da"
 )
 
+var rollappHeightGauge = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
for sake of cleanness I think it's better to define the relevant metrics in a separate file within the manager package or a separate package all together.